### PR TITLE
Britt's About Me

### PR DESCRIPTION
**Pull request by abbritt**
<h1>Added</h1>

- some css to make it pretty
- my face
- a brief about me with links to social media
- link on `index.html`

### CODE REVIEW

File: html/david.html [hunk 1]
```diff
@@ -9,9 +9,9 @@
     <meta name="author" content="Davood">
 
     <!-- CSS Files. Try and have as few of these as possible -->
-    <link rel="stylesheet" href="../css/bootstrap.css" type="text/css">
-    <link rel="stylesheet" href="../css/bootstrap-responsive.css" type="text/css">
-    <link rel="stylesheet" href="../css/david.css" type="text/css">
+    <link rel="stylesheet" href="bootstrap.css" type="text/css">
```
Comment (nw0428) [inline html/david.html]: I think this will make my page not work

File: html/david.html [hunk 2]
```diff
@@ -29,7 +29,7 @@
               <ul class="nav">
                 <li class="active"><a href="index.html">Home</a></li>
                 <li><a href="index.html">Everything David</a></li>
-                <li><a href="index.html">David is Cool</a></li>
+                <li><a href="index.hml">David is Cool</a></li>
```
Comment (nw0428) [inline html/david.html]: Same here

**Review by nw0428 — CHANGES_REQUESTED** [review]
It looks like you may have deleted and or broke my page

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
+.DS_Store
+.DS_Store
```
Comment (nw0428) [inline .gitignore]: Good move here. But we only need this once

File: html/david.html (hunk 2 above)
Comment (abbritt) [reply html/david.html]: huh. looks like I somehow had the broken version of your page on my machine. should be fixed now.

**Comment by abbritt** [comment]
Errors have been fixed.

**Review by nw0428 — APPROVED** [review]

**Pull request merged**